recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "root" %}
 {% set tag_name = "6-32-02" %}
 {% set version = ".".join(tag_name.split("-")|map("int")|map("string")) %}
-{% set build_number = 0 %}
+{% set build_number = 1 %}
```

Comment:
There is no need to bump the build number in this case as we don't want to deploy a new build.

Reply:
Ah fair enough!